core/lock.c

```diff
@@ -74,6 +74,9 @@ int pthread_mutexattr_setprotocol (pthread_mutexattr_t *__attr,
 struct uwsgi_lock_item *uwsgi_lock_fast_init(char *id) {
 
 	pthread_mutexattr_t attr;
+#ifdef EOWNERDEAD
```

Comment:
can't you move it inside if (uwsgi_pthread_robust_mutexes_enabled) block instead?